Add limitation regarding Cypress 11+ for Cypress UI reporting bug (#1759)
saucelabs/sauce-docs: docs/web-apps/automated-testing/cypress.md

We recommend that you avoid the use of special characters when naming your tests. If your test name contains any special characters, your test may not run or its artifacts may not be visible in our platform.
:::

:::caution Cypress 11+

> A bug impacting Cypress 11+ makes Cypress UI to display a browser which is not the one actually used.

Reviewer: ```suggestion
A bug impacting Cypress 11+ is causing the Cypress UI to display a browser that is not the one actually used.
```

Edge is shown on Windows, and Webkit is shown on macOS.

The browser selected in your configuration file is used, and the console log of Cypress is still correct.

> The issue has been reported to cypress.

Reviewer: ```suggestion
The issue has been reported to Cypress.
```
